nautobot_ssot/integrations/infoblox/utils/client.py

```diff
@@ -407,18 +438,18 @@ def delete_network(self, prefix):
         Returns Response:
             {"deleted": "network/ZG5zLm5ldHdvcmskMTkyLjAuMi4wLzI0LzA:192.0.2.0/24/default"}
         """
-        resource = self._get_network_ref(prefix)
+        resource = self._get_network_ref(prefix=prefix, network_view=network_view)
 
         if resource:
             self._delete(resource)
             response = {"deleted": resource}
         else:
             response = {"error": f"{prefix} not found."}
 
-        logger.info(response)
+        logger.debug(response)
```

Comment:
Same as above. Please ensure all debug logging is only showing if debug toggle is True.